doc/programming_model/data_types.md

```diff
@@ -16,6 +16,7 @@ in comparison to fp32.
 | f16       | [IEEE half precision floating-point](https://en.wikipedia.org/wiki/Half-precision_floating-point_format#IEEE_754_half-precision_binary_floating-point_format:_binary16)                 |
 | s8/u8     | signed/unsigned 8-bit integer                                                                                                                                                           |
 | s4/u4     | signed/unsigned 4-bit integer                                                                                                                                                           |
+| s32       | signed/unsigned 32-bit integer                                                                                                                                                          |
```

Comment:
It probably could use a wiki page link. @vpirogov, could you share your thoughts, please?